### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by KayLemba**
In this milestone I did the following:
 
- Create forms to perform the following functions:

 - Create a Post on behalf of the current_user.

 - Create a comment on behalf of the current_user.

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    post = Post.find(params[:id])
+    like = current_user.likes.create(post: post)
+    if like.save!
+      flash[:success] = 'like added'
+    else
+      flash[:error] = 'like was not added'
+    end
+    redirect_to user_post_url
+  end
+end
```
Comment (Emmyn5600) [inline app/controllers/likes_controller.rb:12]: - [x] Good job on implementing the add post and comment form however, there isn't any button or icon to like the post, please work on it.

File: app/views/posts/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <%= form.text_field :title, class: "form-control" %>
+  </div>
+  <div class="mb-3">
+    <%= form.label :Comment %>
+    <%= form.text_area :text, class: "form-control text" %>
+  </div>
+  <div class="mb-3">
+    <%= form.submit class: "btn btn-primary" %>
+  </div>
+<% end %>
```
Comment (Emmyn5600) [inline app/views/posts/new.html.erb:14]: - [x] Optional suggestion 
I would recommend validating the input fields for creating a post and comment.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @koomekevo and @Tumworobere 

Good job so far! I know you have worked hard on making this milestone. There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes :recycle:

- [ ] Check the comment under the review


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (KayLemba) [reply app/controllers/likes_controller.rb:12]: Hello @Emmyn5600 are you referring to me or its the two peoples names "@koomekevo" "@Tumworobere" you are reffering to??

File: app/views/posts/new.html.erb (hunk 2 above)
Comment (KayLemba) [reply app/views/posts/new.html.erb:14]: And I dont understand but what you mean here can you give an example

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @KayLemba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KayLemba** [comment]
Thank you for the review @FrederickMih

**Pull request merged**